lib/pages/add_bridge/add_bridge.dart

```
@@ -176,6 +190,118 @@ class BotController extends State<AddBridge> {
+            if (kDebugMode) {
+              print('Status: Connected to $networkName');
+            }
+          } else {
+            _updateNetworkStatus(network, false, false);
+            if (kDebugMode) {
+              print('Status: Not connected to $networkName');
+            }
+          }
+        } else {
+          // Treat an empty list as unconnected
+          _updateNetworkStatus(network, false, false);
+          if (kDebugMode) {
+            print('Status: Not connected to $networkName (logins list is empty)');
+          }
+        }
+      } else {
+        if (kDebugMode) {
+          print('Network not detected or not supported: $networkName');
+        }
+      }
+    } else {
+      if (kDebugMode) {
+        print('Network not detected in response.');
+      }
+    }
+  }
+
+  String getBridgePath(SocialNetwork network) {
+    switch (network.name) {
+      case 'Facebook Messenger':
+        return 'matrix-mautrix-meta-messenger';
+      default:
+        return 'unknown-bridge';
+    }
+  }
+
+  Future<void> fetchLoginFlows(SocialNetwork network) async {
+    final bridgePathIdentifier  = getBridgePath(network);
+    final accessToken = client.accessToken;
+    final userId = client.userID;
+    final url = Uri.parse(
+      'https://matrix.staging.tawkie.fr/_matrix/$bridgePathIdentifier/provision/v3/login/flows?user_id=$userId',
+    );
+
+    final response = await http.get(
+      url,
+      headers: {'Authorization': 'Bearer $accessToken'},
+    );
+
+    if (response.statusCode == 200) {
+      final responseJson = jsonDecode(response.body);
+      final flows = responseJson['flows'];
+
+      if (flows != null) {
+        print('Available login flows for ${network.name}:');
+        for (var flow in flows) {
+          print('Name: ${flow['name']}, Description: ${flow['description']}, ${flow['id']}');
+        }
+      } else {
+        print('No login flows found for ${network.name}.');
```

This error is not propagated or shown to user